packages/date-picker/src/picker.vue

```diff
@@ -412,8 +423,8 @@ export default {
         this.emitChange(this.value);
         this.userInput = null;
         this.dispatch('ElFormItem', 'el.form.blur');
-        this.$emit('blur', this);
-        this.blur();
+        // this.$emit('blur', this);
```

Comment:
Why is this code commented?

Reply:
This allows the focus to remain on the triggering input text box and next tabbing takes the user to next in line tab index